runtime/Cargo.toml

```diff
@@ -146,7 +147,7 @@ std = [
   "xcm-executor/std",
   "xcm/std",
 ]
-testnet = []
+testnet = ["frame-support-test"]
```

Comment:
`"dep:frame-support-test"`?

Reply:
Only if testnet flag is enabled. The frame_support_test dependency is used. It is used because of [TestRandomness](https://paritytech.github.io/polkadot-sdk/master/frame_support_test/struct.TestRandomness.html)